src/unlock/omni_lock.rs

```diff
@@ -133,6 +136,46 @@ bitflags! {
     }
 }
 
+/// The administrator mode configuration.
+#[derive(Clone, Serialize, Deserialize, Debug, Hash, Eq, PartialEq)]
+pub struct AdminConfig {
+    /// The rc type id hash
+    rc_type_id: H256,
+    /// The smt proofs
+    proofs: Bytes,
```

Comment:
What's the situation that the `proofs` is not `SmtProofEntryVec` is allowed? I suggest the following data type:
```suggestion
    proofs: SmtProofEntryVec,
```

Reply:
use SmtProofEntryVec will need to impl Serialize, Deserialize,

Reply:
For the sake of type safety, we should more Rust data types for following molecule schema:
```protobuf
/* To support multiple RCRules, need to store multiple proofs in every item
in "Bytes structure" in witness.

Which means, one item in "structure" XudtWitnessInput might be SmtProofVec
*/
vector SmtProof <byte>;

/*
mask:
- first bit: use input as target
- second bit: use output as target
if both used, then input and output are combined as target.
if neither used, the corresponding white list is not used, which is OK.

For black list, it always uses both.
*/
table SmtProofEntry {
    mask: byte,
    proof: SmtProof,
}

vector SmtProofEntryVec <SmtProofEntry>;
```